core-common/src/main/java/org/mapstruct/CollectionMappingStrategy.java

```diff
@@ -51,5 +52,16 @@
      * Identical to {@link #SETTER_PREFERRED}, only that adder methods will be preferred over setter methods, if both
      * are present for a given collection-typed property.
      */
-    ADDER_PREFERRED;
+    ADDER_PREFERRED,
+
+    /**
+     * Identical to {@link #SETTER_PREFERRED}, however the target collection will not be cleared and accessed via
```

Comment:
True.. added that.. We should however align our terminology.. I now used the frasing: `updating existing bean instances` which is in line with the documentation.

However, I would like to make some kind of glossary for the terms we introduced.  Update method (or perhaps merge method) would sound better in the long run, don't you think?